src/rule_checker/schema.ts

```diff
@@ -109,7 +110,9 @@ export const validateRuleConfig = (ruleConfig: RuleConfig) => {
         const message = 'validateRuleConfig: Invalid `rules.yaml` config file.';
         console.error(message);
         console.error(ruleConfigValidator.errors);
-        throw TypeError(message);
+        // TODO: look into errors object and see if we can return more
+        // specific error information
```

Comment:
I [created an issue](https://github.com/MikeHopcroft/PrixFixe/issues/100) for this so the TODO doesn't get lost